{
  "comments": [
    {
      "unresolved": false,
      "key": {
        "uuid": "7c451960_2a479484",
        "filename": "/PATCHSET_LEVEL",
        "patchSetId": 1
      },
      "lineNbr": 0,
      "author": {
        "id": 13252
      },
      "writtenOn": "2024-01-11T06:58:05Z",
      "side": 1,
      "message": "recheck volume backup failure in tempest job seems unrelated",
      "revId": "df042959db53703c517156ce8895884a9769d92b",
      "serverId": "4a232e18-c5a9-48ee-94c0-e04e7cca6543"
    }
  ]
}